nnvm/tests/python/compiler/test_to_relay.py

```diff
@@ -0,0 +1,18 @@
+import nnvm
+from nnvm import testing
+from nnvm import to_relay
+
+def check_model(sym, shapes, dtypes):
+    net = nnvm.graph.create(sym)
+    graph_json, mod, params = nnvm.compiler.build(net, 'llvm', shape=shapes, dtype=dtypes)
+    relay_model = to_relay.from_nnvm(net, shapes, dtypes)
```

Comment:
Am I missing an import here? When I try to run this it can't find `from_nnvm` in `to_relay`.